src/proofs/Term.h

```diff
@@ -57,12 +53,12 @@ class Op : public Term {
     std::string simplifyRule();
     bool nonLinearity();
     std::string nonLinearSimplification();
+    void setArg(int i, std::shared_ptr<Term> newArg) { args[i] = std::move(newArg); }
```

Comment:
This method is no longer needed, no?

Reply:
We are still using it, but on a copy, to avoid simplifying the actual term.